package.json

```diff
@@ -98,7 +98,6 @@
     "@storybook/addon-actions": "^6.5.10",
     "@storybook/addon-docs": "^6.5.10",
     "@storybook/addon-links": "^6.5.10",
-    "@storybook/addons": "^6.5.10",
```

Comment:
It will be removed in https://github.com/sendbird/sendbird-uikit-react/pull/1074 PR
I don't recommend editing dependencies now (it cause conflicts)

Reply:
oh, yes, I removed it to run the storybook, let me put it back in this PR